fcrepo-auth-common/src/main/java/org/fcrepo/auth/common/FedoraUserSecurityContext.java

```diff
@@ -15,7 +15,7 @@
  */
 package org.fcrepo.auth.common;
 
-import static org.fcrepo.kernel.api.FedoraJcrTypes.JCR_CONTENT;
+import static org.modeshape.jcr.api.JcrConstants.JCR_CONTENT;
```

Comment:
I disagree. The core issue is this: isolate all modeshape-related code in the `fcrepo-kernel-modeshape` module. And in order to achieve this, there are two primary issues to resolve.

First, this means removing all modeshape-specific code from `fcrepo-kernel-api` where it *most certainly* does not belong. That is the primary goal of this particular PR. And IMO it is the right place to begin.

The second part of the issue relates to removing specific dependencies on `fcrepo-kernel-modeshape`. The primary issue here relates to how `RdfStream::getTriples` is used (See related issue at https://jira.duraspace.org/browse/FCREPO-1312). That is, a class that calls `RdfStream::getTriples` must pass in a class from the `org.fcrepo.kernel.modeshape.rdf.impl` package. See, for example here: http://git.io/v8DUq, where the fcrepo-http-api module is completely tied to the fcrepo-kernel-modeshape module. That sort of interdependence will need to change; but it should be a separate effort (separate from this PR).

In this particular case: fcrepo-auth-common is already entirely bound to modeshape, so using a JCR-related constant from `org.modeshape.jcr.api` instead of `org.fcrepo.kernel.api` makes perfect sense to me.